test/express-example.ts

```diff
@@ -11,6 +12,7 @@ function setupSeneca() {
   Seneca()
     .test()
     .use('repl')
+    .use(flame)
     .use(debug, {
```

Comment:
inside the test code for the current plugin, seneca.use('plugin-name') won't work, so only in this case do you need to manually require